scripts/create_aggregated_zone_config.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+"""
+poetry run python scripts/create_aggregated_zone_config US US-Central
```

Comment:
Can you maybe add to the top comment the exact purpose of that script? For now as the idea is fresh in our minds, it's quite clear, but I don't think we'll be running that script every 2nd week, and we might forget what it's used for.